xbmc/pvr/windows/GUIWindowPVRRecordings.cpp

```diff
@@ -28,6 +28,7 @@
 #include "threads/SingleLock.h"
 #include "utils/URIUtils.h"
 #include "video/windows/GUIWindowVideoNav.h"
+#include "video/VideoLibraryQueue.h"
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers

Reply:
thanks...corrected now